### PR TITLE
NSNG-619-osmani-setup: Copy stex-touch

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        run: npm run generate
+      - name: Copy files via SSH
+        uses: appleboy/scp-action@v0.1.4
+        with:
+          host: webumenia.sk
+          username: lab_sng
+          key: ${{ secrets.DEPLOY_SSH_PRIVATE_KEY }}
+          source: .output/public/*
+          strip_components: 2
+          target: /var/www/stex-touch
```
Comment (igor-kamil) [inline .github/workflows/deploy.yaml:23]:
```suggestion
          target: /var/www/stex.sng.sk/osmani
```

**Review by igor-kamil — COMMENTED** [review]
just update the `deploy.yaml`

File: nuxt.config.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+import svgLoader from 'vite-svg-loader'
+
+export default defineNuxtConfig({
+  modules: ["@nuxtjs/i18n"],
+
```
Comment (igor-kamil) [inline nuxt.config.ts:6]:
```suggestion
  ssr: false,
```

Comment (FrantisekMichalSebestyen) [reply nuxt.config.ts:6]: not needed right? it's same as stex touch no api calling

Comment (igor-kamil) [reply nuxt.config.ts:6]: yes. you are right.

**Review by igor-kamil — APPROVED** [review]

**Pull request merged**